frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { color } from "metabase/lib/colors";
+import {
+  breakpointMinSmall,
+  breakpointMinLarge,
+  breakpointMinMedium,
+} from "metabase/styled-components/theme";
+
+export const Root = styled.div<{
+  hasScroll: boolean;
+  isBordered?: boolean;
+}>`
+  display: flex;
+  flex-direction: column;
+
+  ${props =>
+    props.hasScroll &&
+    css`
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    `}
+
+  ${props =>
+    props.isBordered &&
+    css`
+      border: 1px solid ${color("border")};
+      border-radius: 8px;
+      box-shadow: 0 2px 2px ${color("shadow")};
+    `}
+`;
+
+export const ContentContainer = styled.div<{ hasScroll: boolean }>`
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 auto;
+  position: relative;
+
+  ${props =>
+    props.hasScroll &&
+    css`
+      overflow-y: auto;
+    `}
+`;
+
+export const Header = styled.header`
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem;
+
+  ${breakpointMinSmall} {
+    padding: 1rem;
+  }
+
+  ${breakpointMinLarge} {
+    padding: 1.5rem;
+  }
+`;
+
+export const Body = styled.main`
```

Comment:
I believed this is missing `width: 100%` from `.full`, but I'm not sure if it still behaves the same. Can you help confirm this?

Reply:
You're right! Thanks a bunch